.github/workflows/build.yml

```diff
@@ -62,45 +92,24 @@ jobs:
           docker system df -v
       - name: Export docker container
         run: |
-          docker tag "lfedge/eve:$VERSION-kvm" "$TAG-kvm-${{ env.ARCH }}"
-          docker save "$TAG-kvm-${{ env.ARCH }}" > "eve-kvm-${{ env.ARCH }}.tar"
-      - name: Upload EVE KVM
+          make cache-export IMAGE=lfedge/eve:$VERSION-${{ matrix.hv }} OUTFILE=eve-${{ matrix.hv }}-${{ matrix.arch }}.tar IMAGE_NAME=$TAG-${{ matrix.hv }}-${{ matrix.arch }}
```

Comment:
Will it work without setting of ZARCH to export arm64 image on amd64 host?

Reply:
Haha! That _is_ the error! Going to push now.